TYPO3.TYPO3CR/Classes/TYPO3/TYPO3CR/Utility.php

```diff
@@ -44,6 +47,11 @@ public static function renderValidNodeName($name)
         // Ensure only allowed characters are left
         $name = preg_replace('/[^a-z0-9\-]/', '', $name);
 
+        // Make sure we don't have an empty string left.
+        if ($name === '') {
+            $name = 'node-' . strtolower(md5($originalName));
```

Comment:
so it's using the `node–` convention now, but with full md5 instead of 13 characters..

not sure what the benefit of using md5 on the original name is over a random string, but would work ofc.